packages/ts-morph/src/structures/base/name/PropertyNamedNodeStructure.ts

```diff
@@ -1,3 +1,5 @@
+import { WriterFunction } from "../../../types";
+
 export interface PropertyNamedNodeStructure {
-  name: string;
+  name: string | WriterFunction;
```

Comment:
I'm hesitant to merge this because people are likely doing comparisons on `string`. One of the major issues with structures is that changes that make it easier for writing can make it harder to use for reading. At this point, I think this might be too breaking.